serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-status-conditions.adoc

```diff
@@ -0,0 +1,150 @@
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, status, conditions
+
+This document describes the status and conditions of the `KogitoServerlessWorkflow` object deployed by the {operator_name}.
+
+link:https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties[Kubernetes Status] is an important property to observe in order to understand what is currently happening with the object. It can also help you troubleshoot or integrate with other objects in the cluster. 
+
+== General Status
+
+The table bellow lists the general structure of a workflow status.
+
+.Description of KogitoServerlessWorkflow resource object
+[cols="1,2"]
+|===
+|Status | Description
+
+| `ObservedGeneration`
+| Last object generation observed by the status
+
+| `Conditions`
+| Current observed workflow conditions
+
+| `Address`
+| External or internal workflow access endpoint. Implements the link:https://github.com/knative/specs/blob/main/specs/eventing/overview.md#addressable[Knative Addressable Interface]
+
+| `RecoverFailureAttempts`
+| How many attempts the operator tried to recover from a failure
+
+|===
+
+The `Conditions` property might vary depending on the workflow profile. The next sections describe the current implementation.
+
+== Development Profile Conditions
+
+When you deploy a workflow with the xref:cloud/operator/developing-workflows.adoc[development profile], the operator deploys a ready-to-use container with a running workflow instance. 
+
+The following table lists the possible conditions.
+
+.Conditions Scenarios in Development
+[cols="0,0,1,2"]
+|===
+|Condition | Status | Reason | Description
+
+| Built
+| Unknown
+| 
+| In development profile there's no build process, so this state remains in `Unknown` status
+
+| Running
+| True
+| 
+| The workflow is running and in healthy state
+
+| Running
+| False
+| WaitingForDeployment
+| The workflow is waiting for the underlying deployment object to have a minimum availability
+
+| Running
+| False
+| DeploymentFailure
+| There was a problem with the underlying deployment object. Check the message in this condition and the workflow pod logs for more info
+
+| Running
+| False
+| DeploymentIsUnavailable
+| The underlying deployment object doesn't have the minimum availability for this workflow. Check the message in this condition and the workflow pod logs for more info
+
+| Running
+| False
+| AttemptToRedeployFailed
+| If the workflow deployment is not available, the operator will try to rollout the deployment three times before entering in this stage. Check the message in this condition and the workflow pod logs for more info
+
+|===
+
+In normal conditions the workflow will transition from `Running`, `WaitingForDeployment` condition to `Running`. In case something wrong happens, consult the section xref:cloud/operator/developing-workflows.adoc#troubleshooting[Workflow Troubleshooting in Development].
```

Comment:
```suggestion
In normal conditions, the workflow will transition from `Running`, `WaitingForDeployment` condition to `Running`. In case something wrong happens, consult the section xref:cloud/operator/developing-workflows.adoc#troubleshooting[Workflow Troubleshooting in Development].
```